### ISSUE TITLE
New solvers

### ISSUE THREAD

**Issue by patrick-kidger** [google-research/torchsde#30]
Creating an issue to keep track of the various solvers that would be nice to add (eventually):
**Ito**
- [ ] Derivative-free Milstein [ref](https://infoscience.epfl.ch/record/143450/files/sde_tutorial.pdf)
- [ ] Talay
- [ ] SRK: DRI1 [ref](https://arxiv.org/pdf/1303.5103.pdf)

**Stratonovich**
- [ ] Euler-Heun [ref](https://infoscience.epfl.ch/record/143450/files/sde_tutorial.pdf)
- [ ] Milstein [ref](https://infoscience.epfl.ch/record/143450/files/sde_tutorial.pdf)
- [ ] Derivative-free Milstein [ref](https://infoscience.epfl.ch/record/143450/files/sde_tutorial.pdf)
- [ ] Ninomiya-Victoir [ref](https://arxiv.org/pdf/1612.07017.pdf)
- [ ] SRK: Komori [ref](https://www.sciencedirect.com/science/article/pii/S0377042706003906)

**Comment by lxuechen** [comment]
Adding to this thread I would also advocate for including more high weak order schemes and implicit schemes.

**Comment by patrick-kidger** [comment]
I agree. IIRC both of the SRKs, and NV, are high weak order schemes.
